Gemfile

```diff
@@ -12,12 +12,12 @@ source "https://rubygems.org"
 # If you want to use GitHub Pages, remove the "gem "jekyll"" above and
 # uncomment the line below. To upgrade, run `bundle update github-pages`.
 # all supported plugins by GitHub Pages are included in "github-pages". (GitHub Pages doesn't support 3rd party plugins)
-gem "github-pages", "~> 228"
+gem "github-pages", "~> 229"
 
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
 # and associated library.
 install_if -> { RUBY_PLATFORM =~ %r!mingw|mswin|java! } do
-  gem "tzinfo", "~> 1.2"
+  gem 'tzinfo', '~> 2.0', '>= 2.0.6'
```

Comment:
Why this switch from double to single quotation mark?

Reply:
This is recommended at rubygems page to install tzinfo gem.